integration_test.go

```diff
@@ -368,31 +372,273 @@ func (s *simpleService) Baz(c context.Context, in *simple.Foo) (*simple.Foo, err
 	return nil, fmt.Errorf("negative input")
 }
 
-func (s *simpleService) BidiStream(stream simple.Bar_BidiStreamServer) error {
-	return nil
+func (s *simpleService) ClientStream(stream simple.Bar_ClientStreamServer) error {
```

Comment:
Let's add a comment on what is the behavior of this method. (same comment for ServerStream and BidiStream).
explaining that the return value will be equal to the number of messages received will help for maintaining the test ;)

Reply:
Replaced test service behaviour with simple expected input and output